packages/extension-utils/src/adaptorUtils.ts

```diff
@@ -25,13 +25,24 @@
  */
 
 import type { ThemeCustomizations } from '@looker/design-tokens'
+import type { IAPIMethods } from '@looker/sdk-rtl'
 import { BrowserAdaptor } from './browserAdaptor'
 
+export interface IAuthAdaptor {
+  /** Method for retrieving an instantiated SDK */
+  get sdk(): IAPIMethods
+  /** Method for authenticating against the API server. Auth mechanism is dependent on the authSession implementation
+   * used for the sdk. */
+  login(): void
+}
```

Comment:
This is going to be very useful for customers developing Looker apps with React, I think